### PR TITLE
[codestyle] xml completed

### PR DESCRIPTION
**Pull request by brianteeman**
Looks like there were a few things missed from the monster xml cleanup earlier. This fixes the last few

@quy hope this is ok first time

### CODE REVIEW

File: tests/unit/suites/libraries/cms/installer/data/mod_finder.xml [hunk 1]
```diff
@@ -29,127 +29,131 @@
 					type="searchfilter"
 					default=""
```
Comment (Quy) [inline tests/unit/suites/libraries/cms/installer/data/mod_finder.xml]: Move after `description`.

File: tests/unit/suites/libraries/cms/installer/data/mod_finder.xml [hunk 2]
```diff
@@ -29,127 +29,131 @@
...
 					label="MOD_FINDER_FIELDSET_BASIC_SEARCHFILTER_LABEL"
-					description="MOD_FINDER_FIELDSET_BASIC_SEARCHFILTER_DESCRIPTION" />
+					description="MOD_FINDER_FIELDSET_BASIC_SEARCHFILTER_DESCRIPTION" 
+				/>
+
 				<field
 					name="show_autosuggest"
 					type="list"
 					default="1"
```
Comment (Quy) [inline tests/unit/suites/libraries/cms/installer/data/mod_finder.xml]: Move after `description`.

File: tests/unit/suites/libraries/cms/installer/data/mod_finder.xml [hunk 3]
```diff
@@ -29,127 +29,131 @@
...
+					description="MOD_FINDER_FIELDSET_BASIC_AUTOSUGGEST_DESCRIPTION"
+					>
+					<option value="1">JSHOW</option>
+					<option value="0">JHIDE</option>
 				</field>
+
 				<field
 					name="show_advanced"
 					type="list"
 					default="0"
```
Comment (Quy) [inline tests/unit/suites/libraries/cms/installer/data/mod_finder.xml]: Move after `description`.

File: tests/unit/suites/libraries/cms/installer/data/mod_finder.xml [hunk 4]
```diff
@@ -29,127 +29,131 @@
...
 					type="modulelayout"
 					label="JFIELD_ALT_LAYOUT_LABEL"
-					description="JFIELD_ALT_MODULE_LAYOUT_DESC" />
+					description="JFIELD_ALT_MODULE_LAYOUT_DESC" 
+				/>
+
 				<field
 					name="moduleclass_sfx"
 					type="text"
 					default=""
```
Comment (Quy) [inline tests/unit/suites/libraries/cms/installer/data/mod_finder.xml]: Move after `description`.

**Comment by Quy** [comment]
There are more like the ones mentioned. Do you want me to point out each one?

**Comment by brianteeman** [comment]
Shoot - I just concentrated on the layout and forgot all about the order. I will recheck it all now

**Comment by brianteeman** [comment]
@quy should all be ok now

**Comment by Quy** [comment]
Decided to look at the entire file rather than the changes and found the following to fix:

https://github.com/brianteeman/joomla-cms/blob/ca0d5de6a1d24e950f7672790f91e56ce6a7e685/modules/mod_articles_popular/mod_articles_popular.xml#L46

https://github.com/brianteeman/joomla-cms/blob/ca0d5de6a1d24e950f7672790f91e56ce6a7e685/modules/mod_articles_popular/mod_articles_popular.xml#L111

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 444320e7d5c198c90e321621fdf05ff9bb64c61f<br /><br />Code review.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17315">issues.joomla.org/tracker/joomla-cms/17315</a>.</sub>

**Comment by brianteeman** [comment]
@rdeutz are the appveyor errors something to be concerned abut for this pr?

**Comment by rdeutz** [comment]
seems to me a false positive, have re-started the build

**Comment by brianteeman** [comment]
Can we have one more test please

**Comment by sanderpotjer** [comment]
I have tested this item :white_check_mark: successfully on 444320e7d5c198c90e321621fdf05ff9bb64c61f<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/17315">issues.joomla.org/tracker/joomla-cms/17315</a>.</sub>

**Comment by brianteeman** [comment]
Thanks for testing

**Comment by ghost** [comment]
RTC after two successful tests.

**Comment by brianteeman** [comment]
thanks - glad to see the back of these xml files

**Pull request merged**